src/data/eventregistration.json

```diff
@@ -4,7 +4,7 @@
             "title": "Create QR code for event check-in"
         },
         "intro": "The app now allows a check-in for events and locations. People who host events or have a business can now create a QR code. By scanning the QR code, guests can check in upon arrival to register their presence. If desired, the app will also create a corresponding diary entry. If a checked-in person later tests positive for the coronavirus, other people who were checked in at the same time can be warned.",
-        "introCLI": " You can create QR codes in the app or here on this page. Alternatively, you can also use this command line tool <a href='https://github.com/corona-warn-app/cwa-event-qr-code'>command line tool</a> to generate several QR codes at once, for example.",
+        "introCLI": " You can create QR codes in the app or here on this page. Alternatively, you can also use this <a href='https://github.com/corona-warn-app/cwa-event-qr-code'>command line tool</a> to generate several QR codes at once, for example.",
```

Comment:
Can you change "this command line tool" to "our official QR code utilities"? That should be more precise.

Reply:
Yep, I can do this. I'd also change the German wording then, ok?

Reply:
Yes, please :)

Reply:
Done in [`46472d8` (#1591)](https://github.com/corona-warn-app/cwa-website/pull/1591/commits/46472d81115d8780639bd05e69fae4f94ea7b103).